packages/node/src/models/app-instance.ts

```diff
@@ -78,27 +77,30 @@ export class AppInstance {
     signingKeys: string[],
     defaultTimeout: number,
     appInterface: AppInterface,
-    terms: Terms,
     isVirtualApp: boolean,
     appSeqNo: number,
     rootNonceValue: number,
     latestState: any,
     latestNonce: number,
-    latestTimeout: number
+    latestTimeout: number,
+    beneficiaries: [string, string],
+    limitOrTotal: BigNumber
```

Comment:
Add docstring comments to this class for both `beneficiaries` and `limitOrTotal` please.